tools/Linux/kodi.metainfo.xml.in

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>tv.kodi.Kodi</id>
+    <name>Kodi</name>
+    <project_license>GPL-2.0-only GPL-2.0-or-later LGPL-2.1-or-later MIT BSD-3-Clause BSD-4-Clause</project_license>
+    <metadata_license>CC0-1.0</metadata_license>
+    <developer_name>Team Kodi</developer_name>
+    <summary>The ultimate entertainment center</summary>
+    <url type="homepage">https://kodi.tv/</url>
+    <url type="donation">https://kodi.tv/contribute/donate</url>
+    <url type="bugtracker">https://github.com/xbmc/xbmc/issues</url>
+    <url type="faq">https://kodi.wiki/view/FAQs</url>
+    <url type="help">https://forum.kodi.tv/</url>
+    <url type="translate">https://kodi.weblate.cloud/</url>
+    <description>
+        <p>Kodi spawned from the love of media. It is an entertainment hub
+        that brings all your digital media together into a beautiful
+        and user friendly package. It is 100% free and open source,
+        very customisable and runs on a wide variety of devices. It is
+        supported by a dedicated team of volunteers and a huge community.</p>
+
+        <p>Kodi (formerly known as XBMC) is an award-winning free and
+        open source (GPL) software media player and entertainment hub
+        that can be installed on Linux, OSX, Windows, iOS and Android,
+        featuring a 10-foot user interface for use with televisions and
+        remote controls.</p>
+
+        <p>It allows users to play and view most videos, music, podcasts,
+        and other digital media files from local and network storage media
+        and the internet. Our forums and Wiki are bursting with knowledge
+        and help for the new user right up to the application developer.</p>
+    </description>
```

Comment:
Great description! I would lead with the sentence concerning the "what":

> Kodi allows users to play and view most videos, music, podcasts, and other digital media files from local and network storage media and the internet.

Reply:
Looks great